### PR TITLE
Ruediger's Two In One Extravaganza: Knight Errant Rework + Doctor's Bag Nerf !

### PR DESCRIPTION
**Pull request by Ruediger4**
## About The Pull Request

# UPDATE: KE STAYS, BUT NERFED. CAVALRY IS ITS OWN ADVENTURER CLASS NOW. ANYTHING PAST THIS IS THE OG MSG.

 I shall start by the first thing:

# No adventurer class should start with gear and stats that will make it near impossible to lose a duel without breaking a sweat, let alone make another class irrelevant.

**Knight Errant** has been a topic of mildly hot debate as it is essentially a Knight from before Royal Guard existed, but... for adventurers.

Full plate armor (except one part where the coder fucked up but you still get a hauberk so IDC), a bastard sword, tower shield, somethin like +9 to a variety of stats in total. You can't compete. **WARRIOR** can't compete. Why be a jack of all when you can be *king* of all.

So, here's my counter-offer:

## The Cavalry.

**THREE** (Yes, *three*) subclasses with relatively average gear (leaning towards good), with stats that incentivize frontlining and melee, whilst discourage ranged combat. Also, get a saiga. Genuinely, ride a fucking saiga.

**All of these three subclasses get at least 1 Expert weapon skill, and 1-2 weapon skills on Journeyman or similar as a 'backup' or alternative. They also get ~~EXPERT~~ JOURNEYMAN riding and JOURNEYMAN shield, as well as MEDIUM ARMOR training.**

The point of these new adventurers is to be a middle ground between the other 'combat' classes. *Paladin* is a combat first, support second class. **Barbarians** are brutes that are similar to glass cannons: they hit really hard but usually can't sustain too many wounds if they're placed well. *Warriors* are meant to be jack-of-all. They start with relatively average/mid combat skills in a LOT of weapons, offset by the fact they can wear heavy armor.

Cavalry is meant to fill a gap of that akin to a fighter. Really good at a specific weapon or two, but you need to stick to your shtick, git it? To compensate for the fact you'll be locked in to a specific weapon or two (and possibly saiga-riding), you get shittier INT, PER and SPD stats, but usually a bit more CON and END. STR too, sometimes.

The classes are:
**CATAPHRACT**, a spear-first, sword-second class. Intended to be a spearman atop a horse. Fun fact: Spear + shield is QUITE effective!

**VANGUARD**, a diceroll between onehanded steel swords or a steel axe, with Expert-tier hand-to-hand as well!

and **CONQUEROR**, because flails do not get enough love. If flails aren't your dig, you can buy yourself a mace! However, you're all muscle and little brains.

image

All of the Cavalry Brothers have armor on most of their body, weakpoints alongside the neck, hands and legs. But we both know you'll buy armor anyways.

On the other end of the spectrum: Doctor's Bag finally nerfed. They can only carry surgical tools + things like cloth, water bottles, needles, etc.

Also Cavalry gets their own track! I love spanish guitars.

## Why It's Good For The Game

~~Knight Errant annihilated with a laser from orbit because there's nothing more unfun than spawning with genuinely high end gear as a fuckin adventurer on top of godlike stats.~~

KE stays but lost its spawnstart headstart. You still get good shit. Just less OP stats and less OP gear. Tho NGL you still have full body coverage except ur head. IDK buy a skullcap.

Cavalry is its own thing. Go ham.

Doctor Bag Powercreep Bad.

### CODE REVIEW

**Comment by Ruediger4** [comment]
Forgot to point out that Cavalry will have 6 slots but 100% chance-to-appear rate so people actually test dat shit out!!!

**Comment by Ruediger4** [comment]
> to be fair re:bandit knight that is also exactly what apple wanted them to be which
> 
> eh

# What.

**Comment by Tree445** [comment]
> to be fair re:bandit knight that is also exactly what apple wanted them to be which
> 
> eh

What?

**Comment by Tree445** [comment]
10 and 3 slots are too many, considering there are only 20 adventurer slots total.

**Comment by Ruediger4** [comment]
> 10 and 3 slots are too many, considering there are only 20 adventurer slots total.

Didn't expect everyone to pick them but okay.

4 cavalry and 3 knight sounds good? It's temporary, for testing.

File: code/modules/jobs/job_types/roguetown/adventurer/types/combat/rare/knighterrant.dm [hunk 1]
```diff
@@ -1,46 +1,49 @@
 /datum/advclass/knighterrant
 	name = "Knight-errant"
-	tutorial = "You wandered off from your home seeking adventure, roaming to greener pastures for honor and chilvalry."
+	tutorial = "Once part of a noble family, you were cast away. Yet, your honor remains intact - and your victories many. When a warrior is gone, that will be his best and only bulwark."
```
Comment (Elli-Skala) [inline code/modules/jobs/job_types/roguetown/adventurer/types/combat/rare/knighterrant.dm:3]: Says your honor remains intact in rhe flavor text but there isn't any restraints against picking a inhuman patron, maybe should be abit more vauge?

Comment (Ruediger4) [reply code/modules/jobs/job_types/roguetown/adventurer/types/combat/rare/knighterrant.dm]: Fixed

**Review by Ruediger4 — COMMENTED** [review]
Did it before doing this whoops

**Pull request merged**